llvm/test/MC/AMDGPU/gfx11_asm_vop1.s

```diff
@@ -403,50 +403,50 @@ v_ctz_i32_b32 v5, src_scc
 v_ctz_i32_b32 v255, 0xaf123456
 // GFX11: encoding: [0xff,0x74,0xfe,0x7f,0x56,0x34,0x12,0xaf]
 
-v_cvt_f16_f32 v5, v1
+v_cvt_f16_f32 v5.l, v1
```

Comment:
Just to make sure we are on the same page with this: these patches make no changes to the MC tests, such as reordering of instructions, that would complicate merging them in downstream, right?

Reply:
Since this patch is big, I might do cherry-pick myself so it won't be too complicate for the downstream merging.